pax/plugins/io/Plotting.py

```diff
@@ -179,6 +206,11 @@ def plot_waveform(self, event,
                                       connectionstyle="angle3,"
                                                       "angleA=0,"
                                                       "angleB=-90")
+                if less_black:
+                    #Go to next peak if not the biggest in section, otherwise annotate!
```

Comment:
__[flake8]__
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->